app/middleware/data.ts

```diff
@@ -0,0 +1,39 @@
+import type { Action, Dispatch, Store } from "../reducers/types";
+import { REFRESH_VERDICTS } from "../constants/actionTypes";
+import { VERDICT_SCOPE } from "../reducers/data";
+import { setVerdict } from "../actions/data";
+import { getConfigurationFilterHash } from "../reducers/configuration";
+import { assessFlat } from "../flat/assessment";
+import type { OverviewDataEntry } from "../reducers/data"; // eslint-disable-next-line no-unused-vars
+
+export default ((store: Store) => (next: Dispatch) => async (action: Action) => {
+  if (action.type === REFRESH_VERDICTS) {
+    const {
+      data: {
+        overview,
+        flat,
+        verdicts
+      },
+      configuration
+    } = store.getState();
+
+    if (overview) {
+      const configurationHash = getConfigurationFilterHash(configuration);
+      // $FlowFixMe -- Object.values
```

Comment:
can't remember what `$FlowFixMe` did, either convert to it's TypeScript equivalent or remove

Reply:
First I removed all of them, then I found this. 

In general I feel weird: I have the feeling that doesn't matter what I do, the code just runs :D I have to get more used to the pipeline, understand where errors would pop up if they pop up.